Very intermittent failure of test_progress.TestIntegration_progress

One occurrence of this failure seen in the nightly tests.

```
======================================================================
FAIL: test_linear_progress_triad (test_progress.TestIntegration_progress)
Test that the triad region with the geopm_tprof_post() call reports
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_progress.py", line 244, in test_linear_progress_triad
    err_msg)
  File "./test_progress.py", line 171, in check_progress
    self.assertLess(len(progress) * 0.6, sum(good_idx))
AssertionError: 98.39999999999999 not less than 91

----------------------------------------------------------------------
Ran 51 tests in 1660.651s
```

--- a/integration/test/test_progress.py
+++ b/integration/test/test_progress.py
@@ -160,7 +160,7 @@ class TestIntegration_progress(unittest.TestCase):
         """
         epsilon = 0.05
         abs_epsilon = epsilon * expected_max
-        # Get rid of nan values and values
+        # Get rid of nan values
         good_idx = numpy.isfinite(progress)
         max_prog = max(progress[good_idx])
         util.assertNear(self, expected_max, max_prog, epsilon=epsilon,
@@ -168,7 +168,9 @@ class TestIntegration_progress(unittest.TestCase):
         # Get rid of values at expected_max to avoid bad fit error
         # when completed threads stop increasing progress
         good_idx = good_idx & ~numpy.isin(progress, expected_max)
-        self.assertLess(len(progress) * 0.6, sum(good_idx))
+        # Make sure there are at least 10 samples to over-constrain
+        # the two parameter linear fit.
+        self.assertLess(10, sum(good_idx))
         progress = progress[good_idx]
         time = time[good_idx]
         poly_out = numpy.polyfit(time,

